### ISSUE TITLE
Fonofone dans fonoimage- deux positions, petit ou total

### ISSUE THREAD

**Issue by yves-fonofone** [hub-0-1/fonofone#160]
On élimine donc la représentation sur le côté droit.

Par défaut le fonofone d'une zone accompagne en petit la zone. Si j'appuie sur les flèches d'agrandissement il prend alors tout l'espace, y compris la barre du haut (outils fonoimage).
<img width="1607" alt="Capture d’écran, le 2021-07-20 à 17 27 56" src="https://user-images.githubusercontent.com/80292072/126399204-c7c27911-beb2-4231-bd06-243650a70bd4.png">

**Comment by OlivierRoss** [comment]
Ok

**Comment by yves-fonofone** [comment]
Hum ! Pas tout à fait au point.
1. quand je crée une zone, j'ai fonofone en grand plutôt que le petit format.
2. quand je clique pour le réduire, il faut que je clique 2 fois et ça donne ça :
<img width="396" alt="Capture d’écran, le 2021-08-07 à 23 02 23" src="https://user-images.githubusercontent.com/80292072/128619362-705a3e97-4208-413b-935b-9a3ee76c0521.png">

**Comment by yves-fonofone** [comment]
Oui, mais il va falloir reconfigurer le petit Fonofone. Ça n'a pas de sens, on ne peut pas voir un outil au complet.
Voici ma proposition :
<img width="909" alt="Capture d’écran, le 2021-09-06 à 16 47 46" src="https://user-images.githubusercontent.com/80292072/132260784-f4de4f5c-57ed-47f6-bb4d-9f8510d9f8ae.png">
